.circleci/config.yml

```diff
@@ -284,11 +288,15 @@ commands:
 #        JOBS
 # -------------------------
 jobs:
-  api-build-test:
+  api-unit-test:
```

Comment:
renamed from `api-build-test` to `api-unit-test`

Reply:
nice! @calbach will this break anything?

Reply:
I doubt this will break anything. Outside of this file, these names just affect display strings on PRs and in the Circle UI - there should currently be no other programmatic dependencies on this.